### PR TITLE
feat(github): add bug report template 

### PR DESCRIPTION
**Pull request by laurent-laporte-pro**
Lorsque l'anomalie concerne un bug en recette ou en production, il faudrait les information permettant de diagnostiquer et de reproduire :

* Environnement :
  - [ ] Production
  - [ ] Recette

* Identifiant de l'étude
* Date et heure de l'anomalie (pour rechercher dans les logs)
* Fonction utilisée ou URL de l'API si connue
* Message d'erreur et code d'erreur (500 ?)
* Traceback des logs si connu
* Requête faite sur l'API :

  - URL
  - Données source (input data) : fournir un ZIP si les données sont trop grosses.
  - Données obtenues (actual data),
  - Données attendues (expected data).

### CODE REVIEW

**Comment by hdinia** [comment]
---
name: Bug report
about: Create a report to help us improve
title: ''
labels: ''
assignees: ''

---

## Description

Provide a clear and concise description of the issue.

## Steps to Reproduce

Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

## Expected Behavior

Describe what you expected to happen.

## Actual Behavior

Describe what actually happened.

## Screenshots

If applicable, add screenshots to help explain your problem.

## Possible Solution

If you have a solution in mind, describe it here.

## Environment

- [ ] Production
- [ ] Staging
- [ ] Development

## Date and Time

The date and time when the issue occurred: [e.g. 2021-12-31 12:00 UTC]

## Study ID

If applicable, provide the study ID that this issue relates to: [e.g. 123456789]

## Additional Context

Add any other context about the problem here.

- Browser and version: [e.g. Chrome 88, Safari 14]
- Operating System: [e.g. Windows 10, macOS 11.2]
- Any relevant logs or error messages: [e.g. console outputs, stack traces]
- Any relevant code snippets: [e.g. relevant function calls, configuration files]
- Any relevant library versions: [e.g. React 17.0.1]

**Comment by hdinia** [comment]
Preview of the template @laurent-laporte-pro  @skamril

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+4. See error
+
+## Expected Behavior
+
+Describe what you expected to happen.
+
+## Actual Behavior
+
+Describe what actually happened.
+
```
Comment (laurent-laporte-pro) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Peut-on inverser l'ordre et mettre Actual en premier, puis Expected en deuxième ?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+## Possible Solution
+
+If you have a solution in mind, describe it here.
+
+## Environment
+
+- [ ] Production
+- [ ] Staging
+- [ ] Development
```
Comment (laurent-laporte-pro) [inline .github/ISSUE_TEMPLATE/bug_report.md]: L'option "Development" ne me semble pas pertinent, car si on a un bug, c'est probablement en test unitaires. À discuter...

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+## Date and Time
+
+The date and time when the issue occurred: [e.g. 2021-12-31 12:00 UTC]
+
+## Study ID
+
+If applicable, provide the study ID that this issue relates to: [e.g. 123456789]
```
Comment (laurent-laporte-pro) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Je pense qu'on peut mettre l'ID dans le paragraphe suivant.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## Environment
+
+- [ ] Production
+- [ ] Staging
+- [ ] Development
+
+## Date and Time
+
+The date and time when the issue occurred: [e.g. 2021-12-31 12:00 UTC]
```
Comment (laurent-laporte-pro) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Je pense qu'on peut mettre la date et l'heure dans le paragraphe "Additional Context"

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+## Study ID
+
+If applicable, provide the study ID that this issue relates to: [e.g. 123456789]
+
+## Additional Context
```
Comment (laurent-laporte-pro) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Peut-on renommer en "Additional Information" ? Cela me semble plus général.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
```
Comment (laurent-laporte-pro) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Tu vas supprimer (ou cacher) ce header, n'est-ce pas ?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (skamril) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Add default label

**Review by skamril — CHANGES_REQUESTED** [review]

**Comment by hdinia** [comment]
### Updated

## Description

Provide a clear and concise description of the issue.
## Steps to Reproduce

### Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
## Actual Behavior

Describe what actually happened.
## Expected Behavior

Describe what you expected to happen.
## Screenshots

If applicable, add screenshots to help explain your problem.
## Possible Solution

If you have a solution in mind, describe it here.
## Environment

- [ ] Production
- [ ] Staging
## Additional Information
### Add any other context about the problem here.

- If applicable, provide the study ID that this issue relates to:
- Any relevant logs or error messages: [e.g. console outputs, stack traces]
- Any relevant code snippets: [e.g. relevant function calls, configuration files]
- Any relevant library versions: [e.g. React 17.0.1]
- The date and time when the issue occurred:
- Browser and version: [e.g. Chrome 88, Safari 14]
- Operating System: [e.g. Windows 10, macOS 11.2]

**Pull request merged**